pipeline.d/deploy.yml

```diff
@@ -0,0 +1,54 @@
+deploy:
+  path:
+    - type: habitat
+      name: dev
+      schedule: always
+    - type: environment
+      name: dev
+      filters:
+        family: webui
+      schedule: always
+      sequence: parallel
+    - type: habitat
+      name: test
+      schedule: always
+    - type: environment
+      name: test
+      filters:
+        family: webui
+      schedule: always
+      sequence: parallel
+    - type: habitat
+      name: prod
+      schedule: off_hours
+    - type: environment
+      name: prod
+      filters:
+        family: webui
+      schedule: off_hours
+      sequence: serial
+    - type: habitat
+      name: fedramp-use2-core
+      schedule: off_hours
+  branchPath:
+    - type: habitat
+      name: dev
+      schedule: always
+    - type: environment
+      name: dev
+      filters:
+        family: webui
+      schedule: always
+      sequence: parallel
+    - type: habitat
+      name: test
+      schedule: always
+    - type: environment
+      name: test
+      filters:
+        family: webui
+      schedule: always
+      sequence: parallel
+  changeManagement:
```

Comment:
I just copied from what we had elsewhere, but I think we talked about submitting manually; do we still want these values?